src/settings/components/CreateEditRole/EditRole.js

```diff
@@ -104,21 +104,15 @@ const EditRole = ({ path }) => {
     onClose();
   };
 
-  const isInitialDataReady = !isCapabilityListLoading && isRoleCapabilitySetsLoaded
-  && isInitialRoleCapabilitiesLoaded && !isCapabilitySetsLoading;
+  const isInitialDataReady = isRoleCapabilitySetsLoaded && isInitialRoleCapabilitiesLoaded;
```

Comment:
Petty: Can we name these things the same way if we are checking the same condition? IOW why is `Initial` in one of these names but not the other?

Reply:
Fixed, thank you